app/views/blog/_post_preview.html.erb

```diff
@@ -1,14 +1,17 @@
-<article class="blog">
-  <header>
-    <h2><%= link_to fm[:title], path %></h2>
-    <% if fm[:subtitle].present? %>
-      <div role="doc-subtitle"><%= fm[:subtitle] %></div>
-    <% end %>
+<article class="blog-article">
+  <div>
+    <header>
+      <h2><%= link_to fm[:title], path %></h2>
+      <% if fm[:subtitle].present? %>
```

Comment:
I'm not sure if this is the right point to put this comment - but the subtitle is currently very close to the Published line, rather than the title - possible to change the spacing on this?

![image](https://user-images.githubusercontent.com/87642394/140775540-3a568a6e-8575-46cf-9d65-456ca4a54d48.png)

Reply:
Looks much better now - thanks!